src/styles/index.css

```diff
@@ -1,2 +1,6 @@
 /** FigtreeVariable */
 @import '../../node_modules/@fontsource/figtree/variable.css';
+
+.recharts-surface {
```

Comment:
Instead of this custom css selector that no one will remember I adjusted chart rendering area with margins. `https://github.com/oasisprotocol/explorer/blob/master/src/app/pages/DashboardPage/RoseChartCard.tsx#L70`. Anyway, if it's css selector, margin in each chart or in generic  line chart I don't care but it would be nice to unify this.

Reply:
I didn't find a good option in recharts to add styles to the svg it produced, but it has this class `<svg class="recharts-surface">`. I'd prefer if we didn't need to involve math `margin = lineStrokeWidth + activeDot.r + activeDot.strokeWidth`

Unified? https://github.com/oasisprotocol/explorer/pull/145/commits/79e60a2f51498c21ddd66f4529f14ad61688a68c